docs/codeql/reusables/supported-versions-compilers.rst

```diff
@@ -38,5 +38,5 @@
     .. [7] JSX and Flow code, YAML, JSON, HTML, and XML files may also be analyzed with JavaScript files.
     .. [8] The extractor requires Python 3 to run. To analyze Python 2.7 you should install both versions of Python.
     .. [9] Requires glibc 2.17.
-    .. [10] Swift support is currently in beta. Support for the analysis of Swift 5.4-5.7 requires macOS. Swift 5.7.3 can also be analyzed using Linux.
+    .. [10] Swift support is currently in beta. Support for the analysis of Swift 5.4-5.8.1 requires macOS or Linux.
```

Comment:
Is it true that we can now analyse Swift 5.4 - 5.7.2 on Linux?

Reply:
I believe this the case with the latest fix, and Paolo confirmed that it works.